com.unity.render-pipelines.high-definition/Documentation~/SGNode-Diffusion-Profile.md

```diff
@@ -1,13 +1,21 @@
 # Diffusion Profile Node
 
-The Diffusion Profile Node allows you to sample a [Diffusion Profile](Diffusion-Profile.md) Asset in your Shader Graph. For information on what a Diffusion Profile is and the properties that it contains, see the [Diffusion Profile documentation](Diffusion-Profile.md).
+The Diffusion Profile Node allows you to sample a [Diffusion Profile](https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@latest/index.html?subfolder=/manual/Diffusion-Profile.html) Asset in your Shader Graph. For information on what a Diffusion Profile is and the properties that it contains, see the [Diffusion Profile documentation](https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@latest/index.html?subfolder=/manual/Diffusion-Profile.html).
 
-## Output port
+## Render Pipeline compatability
+
+| **Node**          | **Universal Render Pipeline (URP)** | **High Definition Render Pipeline (HDRP)** |
+| ----------------- | ----------------------------------- | ------------------------------------------ |
+| Diffusion Profile | No                                  | Yes                                        |
+
+## Ports
 
 | name | **Direction** | type | description |
 |--- | --- | --- | --- |
 | **Out** | Output | float | Outputs a unique float that the Shader uses to identify the Diffusion Profile. |
 
+## Remarks
+
```

Comment:
probably just a localization difference, I think that the other SG pages use "Notes" (cause they were written by Americans). Do we care about consistency one way or another here?

Reply:
I made the change. We use Note according to the style guide: https://unity-docs.gitbook.io/style-guide/format/notes-and-info-boxes